app/views/centers/application.html.erb

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
```

Comment:
can remove this? we are already using `app/layouts/application.html.slim`.. and this one isn't used